modules/primer-navigation/lib/underline-nav.scss

```diff
@@ -0,0 +1,79 @@
+.UnderlineNav {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid $gray-200;
+}
+
+.UnderlineNav-items {
+  display: flex;
+  margin-bottom: -1px;
+}
+
+.UnderlineNav-item {
+  padding: $spacer-3 $spacer-2;
+  margin-right: $spacer-3;
+  font-size: $body-font-size;
+  line-height: $lh-default;
+  color: $text-gray;
+  text-align: center;
+  border-bottom: 2px solid transparent;
+
+  .Counter {
```

Comment:
I'd prefer to avoid nesting other component selectors like this because it creates dependencies and means overriding if we want to act differently. We could just use the `ml-1` utility. How strongly do you feel this needs to be built into the pattern?

Reply:
I removed this from the component. My only concern is that people will use inconsistent spacing, but I think it needs to be flexible because markup spacing affects this

Reply:
Yeah this, and the other similar scenarios, are tricky. This is where I feel like we should handle the `Counter` styling in context with the `UnderlineNav` in other parts of the code (say helpers or something), rather than Sass. Handling it in the Sass means adding additional classes to apply to components that may already require a number of classes. That's not the worst but can feel like overkill and be a contributor to CSS bloat. We don't have a good solution for handling elsewhere in the code base right now, but it keeps coming up and is something that's on our radar. If we start to see this particular use case being styled incorrectly, then the best option is to have another class to apply to the `Counter` rather than nesting it.